SingularityExecutor/src/main/java/com/hubspot/singularity/executor/config/SingularityExecutorConfiguration.java

```diff
@@ -218,6 +218,9 @@
   private int dockerClientConnectionPoolSize = 5;
 
   @JsonProperty
+  private int dockerPullRetries = 1;
```

Comment:
does this mean it will attempt once, or retry once? it might make more sense to rename this to something like `maxDockerPullAttempts`